src/ctia/http/routes/observable.clj

```diff
@@ -66,12 +67,17 @@
                          :data
                          first)
              verdict (if (some-> verdict verdict/expired?)
-                       (-> (read-store :judgement
-                                       calculate-verdict
-                                       {:type observable_type
-                                        :value observable_value})
-                           entities/realize-verdict
-                           (#(write-store :verdict create-verdicts [%])))
+                       (when-let [realized-verdict
+                                  (some-> (read-store :judgement
+                                                      calculate-verdict
+                                                      {:type observable_type
+                                                       :value observable_value})
+                                          entities/realize-verdict)]
+                         (log/info "Generated New Verdict:"
+                                   (pr-str realized-verdict))
+                         (write-store :verdict create-verdicts
```

Comment:
We should catch any errors here, and log it, and then continue to return the realized-verdict.  This will allow this end-point to work correctly even when the index goes into a read-only mode, or we have a transient write error.